SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/LayeredLBStrategy.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "LoadBalancing/AbstractLBStrategy.h"
+
+#include "CoreMinimal.h"
+#include "Math/Box2D.h"
+#include "Math/Vector2D.h"
+
+#include "LayeredLBStrategy.generated.h"
+
+class SpatialVirtualWorkerTranslator;
+class UAbstractLockingPolicy;
+
+DECLARE_LOG_CATEGORY_EXTERN(LogLayeredLBStrategy, Log, All)
+
+USTRUCT()
+struct FLBLayerInfo
+{
+	GENERATED_BODY()
+
+	UPROPERTY()
+	FName Name;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLBStrategy> LoadBalanceStrategy;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLockingPolicy> LockingPolicy;
+
+
+	FLBLayerInfo() : Name(NAME_None)
+	{
+	}
+};
+
+/**
+ * A load balancing strategy that ...
```

Comment:
Do we need this comment?

Reply:
Yes, but more importantly, we need for me to have actually added CONTENT to this comment, which I have now done.